### DISCUSSION TITLE
有提现相关的能力

### DISCUSSION THREAD

**Discussion by v012345** [yansongda/pay#537] [Ideas]
虽然提现比支付用到的场景少很多,但是还是有需求的

**Comment by yansongda** [comment]
微信还是支付宝？

**Reply by v012345** [reply]
最近在做聊天软件,有积分提现的要求,支付宝和微信都需要

**Reply by yansongda** [reply]
支付宝 现在就支持的

微信的，好像 微信官方v3版 暂时不支持 体现，没看到相关 api。你也可以帮忙再确认下看看，如果找到的话，可以发下，我到时候看看

**Reply by v012345** [reply]
好的好的

**Reply by v012345** [reply]
https://pay.weixin.qq.com/wiki/doc/api/tools/mch_pay.php?chapter=14_1

应该是从"企业对公账户转到微信零钱"的文档
好像是不支持个人和个体工商户的

**Comment by yorkgometer** [comment]
thinkphp5.1报错如下：
Return value of Yansongda\Pay\Pay::getContainer() must be an instance of Psr\Container\ContainerInterface, instance of think\App returned


麻烦请问文档中// $container = 您现有的 container中的"您现有的 container"是什么容器？
